exporter/azuremonitorexporter/config.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azuremonitorexporter
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+)
+
+// Config defines configuration for Azure Monitor
+type Config struct {
+	// squash ensures fields are correctly decoded in embedded struct.
+	configmodels.ExporterSettings `mapstructure:",squash"`
+	Endpoint                      string `mapstructure:"endpoint"`
+	InstrumentationKey            string `mapstructure:"instrumentation_key"`
+	MaxBatchSize                  int    `mapstructure:"maxbatchsize"`
+	MaxBatchIntervalInSeconds     int    `mapstructure:"maxbatchinterval_sec"`
```

Comment:
Up to you: in general we go with `MaxBatchInterval time.Duration ` and `mapstructure:"maxbatchinterval"` so the user can specify the unit on the config, eg.: "5s", "1m", etc.

Reply:
I will do this.